sw/airborne/modules/nav/nav_rotorcraft_hybrid.c

```diff
@@ -57,7 +57,23 @@ float nav_goto_max_speed = NAV_HYBRID_GOTO_MAX_SPEED;
 #define NAV_HYBRID_MAX_DECELERATION 1.0
 #endif
 
-float nav_max_deceleration_sp = NAV_HYBRID_MAX_DECELERATION;
+#ifndef NAV_HYBRID_MAX_ACCELERATION
+#define NAV_HYBRID_MAX_ACCELERATION 4.0
+#endif
+
+#ifndef NAV_HYBRID_SOFT_ACCELERATION
+#define NAV_HYBRID_SOFT_ACCELERATION NAV_HYBRID_MAX_ACCELERATION
+#endif
+
+float nav_max_deceleration_sp      = NAV_HYBRID_MAX_DECELERATION;  //Maximum deceleration allowed for the set-point
+float nav_max_acceleration_sp      = NAV_HYBRID_MAX_ACCELERATION;  //Maximum acceleration allowed for the set-point
+float nav_hybrid_soft_acceleration = NAV_HYBRID_SOFT_ACCELERATION; //Soft acceleration limit allowed for the set-point (Equal to the maximum acceleration by default)
```

Comment:
If ```nav_hybrid_max_acceleration``` is for circles and ```nav_hybrid_soft_acceleration``` is for lines, it would be better if the names reflect this.

Reply:
I would not say that they are meant to be used either for lines or for circles. For example nav_hybrid_soft_acceleration is used in circles when we are far away from it. With these defines we can design a part of a navigation function to have aggressive characteristics, while another part can have more gentle ones.